docs/manage/fleet/system-settings.md

```diff
@@ -0,0 +1,199 @@
+---
+linkTitle: "Configure machine settings"
+title: "Configure machine operating system settings"
+weight: 50
+layout: "docs"
+type: "docs"
+description: "Configure network settings, operating system package updates and logging defaults."
+---
+
+The `viam-agent` configuration allows you to configure:
+
+- [settings for package updates for the host operating system](#configure-os-package-updates)
+- [networks a machine can connect to](#configure-networks)
+- [parameters for operating system logging](#configure-operating-system-logging)
+
+## Configure OS package updates
```

Comment:
I'm unclear why a single field is explained in full three different times, with full json examples for each, when all that changes is one word. Wouldn't it be a lot easier to understand if the field was explained, and then just state that there are three options for "type"?